### PR TITLE
MTA-1181: Attributes Update MTA 7.0.0

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira 

* [MTA-1181](https://issues.redhat.com/browse/MTA-1181)

### Updates 

```
:DocInfoProductNumber: 7.0
:ProductShortName: MTA
...
:ProductVersion: 7.0,0
...
:ProductDistributionVersion: 7.0.0.GA-redhat
:ProductDistribution: mta-7.0.0.GA-cli-offline.zip
```

### CODE REVIEW

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -22,11 +22,11 @@ ifdef::mta[]
 :WebNameTitle: User Interface
 :WebNameURL: user_interface_guide
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.2.1
+:ProductVersion: 7.0,0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.2.1.GA-redhat-00002
-:ProductDistributionVersion: 6.2.1.GA-redhat
-:ProductDistribution: mta-6.2.1.GA-cli-offline.zip
+:MavenProductVersion: 7.0.0.GA-redhat-00001
```
Comment (dymurray) [inline docs/topics/templates/document-attributes.adoc:27]: I don't think this is relevant anymore, right? We don't ship anything in Maven

Comment (anarnold97) [reply docs/topics/templates/document-attributes.adoc:27]: Cool, so should I remove [Maven Plugin Guide](https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.2/html/maven_plugin_guide/index) for 7.0
Thanks

File: docs/topics/templates/document-attributes.adoc [hunk 2]
```diff
@@ -22,11 +22,11 @@ ifdef::mta[]
...
 :WebConsoleBookName: {WebNameTitle} Guide
-:ProductVersion: 6.2.1
+:ProductVersion: 7.0,0
 :PluginName: MTA plugin
-:MavenProductVersion: 6.2.1.GA-redhat-00002
-:ProductDistributionVersion: 6.2.1.GA-redhat
-:ProductDistribution: mta-6.2.1.GA-cli-offline.zip
+// :MavenProductVersion: 7.0.0.GA-redhat-00001
+:ProductDistributionVersion: 7.0.0.GA-redhat
+:ProductDistribution: mta-7.0.0.GA-cli-offline.zip
```
Comment (sshveta) [inline docs/topics/templates/document-attributes.adoc:29]: I don't think we are distributing CLI like this . 

[linux-mta-cli.zip](http://download.lab.bos.redhat.com/rcm-guest/staging/jboss-migrationtoolkit/MTA-7.0.0/linux-mta-cli.zip)

Comment (anarnold97) [reply docs/topics/templates/document-attributes.adoc:29]: ProductDistribution is going to be redundant for MTA

ProductDistributionVersion needs two minor changes that i will do later 

See JIRA for details

**Pull request merged**